jujugui/static/gui/src/app/components/user-profile/user-profile.js

```diff
@@ -283,11 +285,21 @@ YUI.add('user-profile', function() {
     */
     _generateModelRow: function(model) {
       var uuid = model.uuid;
+      var isCurrent = model.name === this.props.currentModel;
+      if (!model.isAlive) {
```

Comment:
But do we have a test for this logic? If not, please add a test for zombie models.